script1.js

```diff
@@ -0,0 +1,26 @@
+var cssInput = document.querySelector('#cssInput');
+cssInput.addEventListener('input', (e) => {
+  styleParser(e.target.value);
+})
+function styleParser(text) {
```

Comment:
While I'm impressed that you took the time to create your own text parser to create valid CSS from the text that is input into the textarea, it IS possible to use a <style></style> tag in your HTML body.  You can take the entire string value of the textarea and insert it in the style tags by either targeting an existing set of style tags or appending them to the document body.

Reply:
I'm also not sure that the approach you took will work on anything that requires cascading styles or specificity.  I'd have to think further on that one though.  Again, kudos to the effort involved.

Reply:
LOL, yes, good point. Thanks for the reference.